tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -4,19 +4,19 @@ def build_empty_execution_payload(spec, state):
     """
     latest = state.latest_execution_payload_header
     timestamp = spec.compute_time_at_slot(state, state.slot)
-    empty_txs = spec.List[spec.OpaqueTransaction, spec.MAX_EXECUTION_TRANSACTIONS]()
+    empty_txs = spec.List[spec.Transaction, spec.MAX_TRANSACTIONS_PER_PAYLOAD]()
 
     payload = spec.ExecutionPayload(
-        block_hash=spec.Hash32(),
         parent_hash=latest.block_hash,
         coinbase=spec.Bytes20(),
         state_root=latest.state_root,  # no changes to the state
-        number=latest.number + 1,
+        logs_bloom=spec.ByteVector[spec.BYTES_PER_LOGS_BLOOM](),  # TODO: zeroed logs bloom for empty logs ok?
```

Comment:
All zeroes is what you get if there were no logs. I think this TODO can be removed

Reply:
Heads up: leaving it in for the purpose of this PR.